ckcp/gitops.sh

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env bash
+
+#quit if exit status of any cmd is a non-zero value
+set -euo pipefail
+# set -x
+
+SCRIPT_DIR="$(cd "$(dirname "$0")" >/dev/null ; pwd)"
+GITOPS_DIR="$(dirname "$SCRIPT_DIR")/gitops"
+KUBECONFIG_CLUSTER="${KUBECONFIG:-$HOME/.kube/config}"
+
+
+
+install_openshift_gitops(){
+  export KUBECONFIG="$KUBECONFIG_CLUSTER"
+
+  local ns="openshift-operators"
+
+  #############################################################################
+  # Install the gitops operator
+  #############################################################################
+  echo -n "  - OpenShift-GitOps: "
+  if ! oc get subscriptions -n "$ns" openshift-gitops-operator >/dev/null 2>&1; then
+    oc apply -f "$GITOPS_DIR/$APP.yaml" --wait >/dev/null
```

Comment:
APP variable is global, but it's always the same in this function. To keep the function independent I would recommend to set it as local variable to be independent.
Currently to run this function this has to be done:
```
APP=openshift-gitops
install_openshift_gitops
```

Reply:
Done